Core/MethodArguments/MethodArgumentFromInvocationNullDocumentConverter.cs

```diff
@@ -17,21 +17,27 @@
 using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.CSharp.Syntax;
 
-namespace NullableReferenceTypesRewriter.ConsoleApplication.MethodArguments
+namespace NullableReferenceTypesRewriter.MethodArguments
 {
+  /// <summary>
+  /// A DocumentConverter, which annotates methodArguments as nullable, if the method is called with null parameters.
+  /// Note: due to the implementation of DocumentConverters it is currently not possible to track methodcalls across documents (classes)

```

Comment:
This could be a `<remarks>` block.

```suggestion
  /// Due to the implementation of DocumentConverters it is currently not possible to track method calls across documents (classes).
```

Add a `see` for "DocumentConverters" if possible.